<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des Equipements</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajouter Equipement</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>511</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Ajout</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nomE">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_qte">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Afficher Equipement</string>
      </attribute>
      <widget class="QTableView" name="tab_equipement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>421</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_searched_eq">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_search_eq">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_exportpdf_eq">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Export PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Supprimer Equipement</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_supp">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Modifier Equipement</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>511</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Modifier</string>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nouveau Nom </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nouvelle Quantité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nomE_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_qte_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_e">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestion des vehicules</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>291</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Ajouter Vehicule</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>511</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Ajout</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Marque</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Modèle</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_mat">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_mar">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_v">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_eta">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_qtev">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Afficher Vehicule</string>
      </attribute>
      <widget class="QTableView" name="tab_vehicule">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>421</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_exportpdf_v">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Export PDF</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_searched_v">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_search_v">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Supprimer Vehicule</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mat_supp">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer_v">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Modifier Vehicule</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>511</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Modifier</string>
       </property>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifer Etat </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier Quantité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_mat_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_etat_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_qtev_mod">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_e_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
